microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md

```diff
@@ -156,18 +156,29 @@ eDiscovery Case Hold Policy Sync Distribution error. The error reads:
 
 ### Resolution
 
-1.	Connect to [Office 365 Security & Compliance Center PowerShell](https://docs.microsoft.com/powershell/exchange/office-365-scc/connect-to-scc-powershell/connect-to-scc-powershell) and then enter the following command:
+1.	Connect to [Office 365 Security & Compliance Center PowerShell](https://docs.microsoft.com/powershell/exchange/office-365-scc/connect-to-scc-powershell/connect-to-scc-powershell) and then enter the following command for eDiscovery CaseHold
+
+ ```powershell
+    Get-CaseHoldPolicy  <policyname> - DistributionDetail | FL
+    ```
+For a Retention Policy Hold , run the following:
 
     ```powershell
     Get-RetentionCompliancePolicy  <policyname> - DistributionDetail | FL
     ```
 
 2. Examine the value in the DistributionDetail parameter for errors like the following:
+ 
+   > Error: Resources :It's taking longer than expected to deploy the policy. It might take an additional 2 hours to update the final deployment status, so check back in a couple hours.” 
```

Comment:
```suggestion
   > Error: Resources: It's taking longer than expected to deploy the policy. It might take an additional two hours to update the final deployment status, so check back in a few hours.” 
```